components/frontend/src/metric/Metric.test.js

```diff
@@ -84,3 +84,22 @@ it('renders the minutes as percentage', () => {
     expect(screen.getAllByText(/50% minutes/).length).toBe(1);
     expect(screen.getAllByText(/≦ 0% minutes/).length).toBe(1);
 });
+
+it('renders correct popups with status', async () => {
+    render_metric("violations");
+    fireEvent.mouseOver(screen.getByText(/50 violations/));
+
+    await waitFor(() => screen.getByTestId('value-popup'))
+    expect(screen.getAllByText(/Metric was last measured/).length).toBe(1);
+    expect(screen.getAllByText(/Value was first measured/).length).toBe(1);
+});
+
+it('renders correct popups without status', async () => {
+    report.subjects.subject_uuid.metrics.violations.status = null;
+    render_metric("violations");
+    fireEvent.mouseOver(screen.getByText(/50 violations/));
+
+    await waitFor(() => screen.getByTestId('value-popup'))
```

Comment:
I'm not a fan of data-testid. Can't we wait for screen.getByText(/Last measurement attempt/)?